main.go

```diff
@@ -8,13 +8,16 @@ import (
 	"github.com/coreos/fleet/schema"
 )
 
+//TODO add env var support
 var (
 	destroyFlag             = flag.Bool("destroy", false, "Destroy units not found in the definition")
 	fleetEndpoint           = flag.String("fleetEndpoint", "", "Fleet API http endpoint: `http://host:port`")
 	socksProxy              = flag.String("socksProxy", "", "address of socks proxy, e.g., 127.0.0.1:9050")
 	destroyServiceBlacklist = map[string]struct{}{"deployer.service": struct{}{}, "deployer.timer": struct{}{}}
 	rootURI                 = flag.String("rootURI", "", "Base uri to use when constructing service file URI. Only used if service file URI is relative.")
 	isDebug                 = flag.Bool("isDebug", false, "Enable to show debug logs.")
+	etcdURL                 = flag.String("etcd-url", "http://localhost:2379", "etcd URL")
+	gtgURL                  = flag.String("gtg-url", "http://localhost:8080", "gtg URL")
```

Comment:
I think a better name would be `gtgBaseUrl` as this will be used to construct the _actual_ gtg url...